src/kili/services/copy_project/__init__.py

```diff
@@ -185,7 +184,12 @@ def download_and_upload_assets(assets):
                 downloaded_assets = self._download_assets(from_project_id, fields, tmp_dir, assets)
                 return self._upload_assets(new_project_id, downloaded_assets)
 
-        return AssetQuery(self.kili.auth.client)(where, fields, options, download_and_upload_assets)
+        asset_gen = AssetQuery(self.kili.auth.client)(
```

Comment:
not returning AssetQuery does not break anything ?

Reply:
we don't use the result of _copy_assets():

```python
        if copy_assets:
            logger.info("Copying assets...")
            self._copy_assets(from_project_id, new_project_id)
```

Reply:
otherwise you can directly write `asset = list(AssetQuery(self.kili.auth.client)(where, fields, options, download_and_upload_assets))`